etcd/getting-started-with-etcd.md

```diff
@@ -6,6 +6,36 @@ Application containers running on your cluster can read and write data into etcd
 
 <a class="btn btn-default" href="../latest/api.html">Complete etcd API Docs</a>
 
+## Setting up etcd
+
+Container Linux comes pre-installed with an `etcd-wrapper` script which can be configured to **run etcd from inside of a container**. Container Linux also comes pre-installed with an `etcd-member.service` systemd unit which by default uses the etcd3 image. This means you to use etcd3 without needing to install the etcd3 binary.
```

Comment:
"Container Linux's `etcd-member.service` systemd unit knows how to fetch and run the current etcd v3.*x* container image, providing etcd v3 without requiring the binary to be present in every default OS installation."